h2o-hadoop/h2o-mapreduce-generic/src/main/java/water/hadoop/h2odriver.java

```diff
@@ -1185,6 +1185,11 @@ private int run2(String[] args) throws Exception {
       conf.set("mapred.job.reuse.jvm.num.tasks", "1");
     }
 
+    // don't kill the the job if we're exiting the driver
+    if (disown) {
+      conf.set("mapred.jobclient.killjob.onexit", "false");
```

Comment:
Just out of curiosity: I didn't find this property documented anywhere. Is it distribution specific? Can you please point me to the documentation?